CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 All notable changes to this project will be documented in this file.
 
+## [[8.6.0]](https://github.com/iExecBlockchainComputing/tee-worker-pre-compute/releases/tag/vNEXT) 2024-12-20
```

Comment:
```suggestion
## [[8.6.0]](https://github.com/iExecBlockchainComputing/tee-worker-pre-compute/releases/tag/v8.6.0) 2024-12-20
```

Reply:
resolved in commit [2801494](https://github.com/iExecBlockchainComputing/tee-worker-pre-compute/pull/93/commits/280149463d507a8704a4b6175de08419270fab3e)